dunglas/doctrine-json-odm/0.1/post-install.txt

```diff
@@ -0,0 +1,6 @@
+<bg=blue;fg=white>              </>
+<bg=blue;fg=white> What's next? </>
+<bg=blue;fg=white>              </>
+
+  * The <comment>json_document</> column type is now available
+  * Go to https://github.com/dunglas/doctrine-json-odm for examples and docs
```

Comment:
I would remove this post install as it does not give any useful information the user won't already have.

Reply:
done